questions.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+var questions = [],
+    fs = require('fs'),
+	JSON5 = require('json5'),
+    correctNameEnding,
+    jsonData;
+
+var originCountries = ['Україна', 'Англія', 'Франція', 'Іспанія', 'Німеччина', 'Чехія', 'Молдова', 'Уельс', 'Китай', 'Азербайджан', 'Чилі'];
+
+//fs.readFile('./public/freebase_countries.json', 'utf8', function(err, data){
+//    jsonData = JSON5.parse(data);
+//    getDataFromJson(jsonData, questions);
+//    console.log(questionsJS.questions);
+//});
+
+for(var i in originCountries) {
+    correctNameEnding = changeCountryNameEnding(originCountries[i]);
+    questions.push(correctNameEnding);
+}
```

Comment:
Yes.
1) its a put a function. makeRightEndings its whole function, we didnt call it here
2 and 3 variant are the same, but in 2 we create function inside the map
in 3 variand we create function and then put it on map and call it there. makeRightEndings will return something.